chainlink-testing-framework/run-tests/action.yml

```diff
@@ -88,7 +88,7 @@ runs:
     # Setup Tools and libraries
     - name: Setup environment
       if: inputs.run_setup == 'true'
-      uses: smartcontractkit/chainlink-github-actions/chainlink-testing-framework/setup-run-tests-environment@v2.1.7
+      uses: smartcontractkit/chainlink-github-actions/chainlink-testing-framework/setup-run-tests-environment@58bbaeb58272dda95d9ee4b98a8406ee72885396
```

Comment:
You will want to change this to the version that will be available when you make it.